enhancements/declarative-index-config.md

```diff
@@ -69,11 +69,12 @@ A human consumable representation of packages in an index in the form of json/ya
 As a user with pull permission from the namespace an index is hosted in/as a component of OLM, I can query an index for a json/yaml representation of the packages of an index.
 The representation of individual packages will allow individual package owners to reason about/make changes to their individual packages in isolation.  
 
-`opm index inspect --image=docker.io/my-namespace/community-operators --output=json`
+`opm unpack docker.io/my-namespace/community-operators`
 
 pulls the index image from the registry, creates a new folder `community-operators` and copies the package representations for the packages that are in the image to the local folder. 
 
 ```bash
+$ opm unpack docker.io/my-namespace/community-operators
```

Comment:
It is a bit weird that we use those commands up here on the top but at the bottom we actually introduce the commands and describe what they do. I guess it is the artifact of the enhancement structure.

Reply:
Gotta write the stories first? 🤷🏽‍♂️

Reply:
Well, the story doesn't have to mention the command specifically. It can simply describe the behavior. Anyway, it is not a big deal.